src/main/java/org/opentripplanner/model/modes/AllowTransitModeFilter.java

```diff
@@ -14,6 +14,7 @@ static AllowTransitModeFilter of(Collection<MainAndSubMode> modes) {
     return FilterFactory.create(modes);
   }
 
+  // TODO: 2022-12-05 this is not the best name since this interface is used both for whitelist and blacklist
```

Comment:
I think you may rename to something like `match` if you want.

Reply:
Renamed